checkROI: 5
pCamLeft:
  euler: [59.9916, 0.026, -177.9387]
  pos: [39.9923, 380.0768, 559.8448]
pCamRight:
  euler: [59.9916, 0.044, -177.9912]
  pos: [-120.0479, 370.8868, 559.4956]
pCenter:
  euler: [59.9916, 0.0128, -177.9206]
  pos: [-33.3959, 370.6583, 508.4605]
pFront:
  euler: [59.9916, 0.0713, -177.9113]
  pos: [259.9762, 244.9297, 349.8502]
pHead:
  euler: [59.9916, 0.0602, -177.9193]
  pos: [260.0092, 304.963, 349.8579]
pHome:
  euler: [59.9916, -0.2716, 179.9456]
  pos: [0.0362, 234.9968, 509.4137]
pLeftWing:
  euler: [59.9916, 0.0474, -177.916]
  pos: [214.9953, 174.9472, 349.9447]
pRear:
  euler: [59.9916, 0.0529, -177.9057]
  pos: [259.9669, 99.9717, 349.92]
pRightWing:
  euler: [59.9916, 0.063, -177.9056]
  pos: [294.9806, 174.9479, 349.8788]
pSuction:
  euler: [59.9916, 0.0355, -177.9437]
  pos: [39.9826, 379.9883, 419.8706]
pTail:
  euler: [59.9916, 0.045, -177.9071]
  pos: [259.9765, 29.9965, 349.9609]
pixelRate: 0.3871875
rollObject: {Front: 59.9916, Head: 59.9916, LeftWing: 59.9916, Rear: 59.9916, RightWing: 59.9916,
  Tail: 59.9916}
score_threshold: 0.5
